Keycloak/grad-client-scopes.lst

```diff
@@ -1,97 +1,4007 @@
-READ_GRAD_DOCUMENT_STATUS_CODE_DATA
-RUN_RULE_ENGINE
-READ_GRAD_AND_PEN_STUDENT_DATA
-READ_GRAD_COURSE_REQUIREMENT_DATA
-READ_SIGNATURE_IMAGE_BY_CODE
-READ_GRAD_CERTIFICATE_CODE_DATA
-UPDATE_GRAD_PROGRAM_SETS_DATA
-DELETE_GRAD_UNGRAD_CODE_DATA
-READ_GRAD_MESSAGING_CODE_DATA
-READ_GRAD_ALGORITHM_RULES_DATA
-READ_GRAD_COUNTRY_CODE_DATA
-UPDATE_GRAD_STUDENT_REPORT_DATA
-READ_GRAD_SPECIAL_CASE_DATA
-READ_GRAD_GRADUATION_STATUS
-CREATE_GRAD_STUDENT_NOTES_DATA
-DELETE_GRAD_PROGRAM_CODE_DATA
-DELETE_GRAD_STUDENT_STATUS_CODE_DATA
-READ_GRAD_PROGRAM_CODE_DATA
-LOAD_BATCH_DASHBOARD
-READ_SIGNATURE_BLOCK_TYPE_CODE
-UPDATE_GRAD_STUDENT_SPECIAL_DATA
-CREATE_PACKING_SLIP
-LOAD_STUDENT_IDS
-READ_GRAD_STUDENT_COURSE_DATA
-CREATE_STUDENT_TRANSCRIPT_REPORT
-READ_GRAD_UNGRAD_CODE_DATA
-CREATE_GRAD_UNGRAD_CODE_DATA
-READ_GRAD_STUDENT_DATA
-READ_GRAD_REQUIREMENT_TYPE_CODE_DATA
-UPDATE_GRAD_PROGRAM_RULES_DATA
-DELETE_GRAD_CERTIFICATE_CODE_DATA
-CREATE_GRAD_PROGRAM_TYPE_CODE_DATA
-CREATE_SCHOOL_DISTRIBUTION
-CREATE_GRAD_PROGRAM_CODE_DATA
-READ_GRAD_LETTER_GRADE_DATA
-DELETE_GRAD_REPORT_CODE_DATA
-READ_GRAD_ASSESSMENT_DATA
-CREATE_STUDENT_CERTIFICATE
-READ_GRAD_STUDENT_STATUS_CODE_DATA
-UPDATE_GRAD_COURSE_RESTRICTION_DATA
-CREATE_GRAD_CAREER_PROGRAM_CODE_DATA
-READ_GRAD_PROGRAM_TYPE_CODE_DATA
-UPDATE_GRAD_STUDENT_CERTIFICATE_DATA
-UPDATE_GRAD_PROGRAM_TYPE_CODE_DATA
-CREATE_STUDENT_XML_TRANSCRIPT_REPORT
-READ_GRAD_SPECIAL_PROGRAM_RULES_DATA
-READ_GRAD_PROVINCE_CODE_DATA
-DELETE_GRAD_CAREER_PROGRAM_CODE_DATA
-CREATE_STUDENT_ACHIEVEMENT_REPORT
-READ_GRAD_STUDENT_REPORT_DATA
-UPDATE_GRAD_UNGRAD_CODE_DATA
-DELETE_GRAD_PROGRAM_TYPE_CODE_DATA
-READ_GRAD_STUDENT_ASSESSMENT_DATA
-UPDATE_GRAD_REPORT_CODE_DATA
-DELETE_GRAD_PROGRAM_RULES_DATA
-READ_GRAD_TRANSCRIPT_CODE_DATA
-READ_GRAD_STUDENT_CERTIFICATE_DATA
-UPDATE_GRAD_STUDENT_NOTES_DATA
-READ_GRAD_HISTORY_ACTIVITY_CODE_DATA
-UPDATE_GRAD_PROGRAM_CODE_DATA
-READ_GRAD_PSI_DATA
-CREATE_GRAD_PROGRAM_RULES_DATA
-READ_GRAD_STUDENT_SPECIAL_DATA
-READ_GRAD_STUDENT_CAREER_DATA
-DELETE_GRAD_STUDENT_NOTES_DATA
-READ_GRAD_REPORT_CODE_DATA
-CREATE_GRAD_CERTIFICATE_CODE_DATA
-UPDATE_GRAD_CAREER_PROGRAM_CODE_DATA
-UPDATE_GRAD_CERTIFICATE_CODE_DATA
-CREATE_OR_UPDATE_SIGNATURE_IMAGE
-READ_GRAD_SPECIAL_PROGRAM_CODE_DATA
-READ_GRAD_COURSE_RESTRICTION_DATA
-READ_GRAD_COURSE_DATA
-RUN_GRAD_ALGORITHM
-CREATE_GRAD_REPORT_CODE_DATA
-UPDATE_GRAD_STUDENT_STATUS_CODE_DATA
-CREATE_GRAD_SPECIAL_PROGRAM_RULES_DATA
-CREATE_GRAD_SPECIAL_PROGRAM_CODE_DATA
-CREATE_GRAD_ASSESSMENT_REQUIREMENT_DATA
-READ_GRAD_STUDENT_UNGRAD_REASONS_DATA
-CREATE_GRAD_STUDENT_UNGRAD_REASONS_DATA
-DELETE_GRAD_SPECIAL_PROGRAM_RULES_DATA
-DELETE_GRAD_REQUIREMENT_TYPE_CODE_DATA
-READ_GRAD_ASSESSMENT_REQUIREMENT_DATA
-UPDATE_GRAD_SPECIAL_PROGRAM_RULES_DATA
-UPDATE_GRAD_SPECIAL_PROGRAM_CODE_DATA
-CREATE_OR_UPDATE_SIGNATURE_BLOCK_TYPE_CODE
-CREATE_GRAD_REQUIREMENT_TYPE_CODE_DATA
-UPDATE_GRAD_REQUIREMENT_TYPE_CODE_DATA
-DELETE_GRAD_SPECIAL_PROGRAM_CODE_DATA
+[
+  {
```

Comment:
We only need GRAD scopes. This file includes ALL scopes including system level scopes. Please remove any scopes that are not required by GRAD.